dhall/src/Dhall/Repl.hs

```diff
@@ -506,8 +572,8 @@ completeFunc reversedPrev word
 
 
 greeter :: MonadIO m => m ()
-greeter =
-  return ()
+greeter = do
+  liftIO (putStrLn "Type :help for more information.")
```

Comment:
Your issue mentions showing the version of Dhall on greeting. I think that's a great idea! :smile: 

Here's how we are getting the version of `dhall-haskell` for the command line interface if that helps:
https://github.com/dhall-lang/dhall-haskell/blob/8c3db821dbc0ad8cfd01d09a58e88126bda717ab/dhall/src/Dhall/Main.hs#L71
https://github.com/dhall-lang/dhall-haskell/blob/8c3db821dbc0ad8cfd01d09a58e88126bda717ab/dhall/src/Dhall/Main.hs#L359